service/matching/matchingEngine.go

```diff
@@ -351,6 +366,21 @@ func (e *matchingEngineImpl) AddActivityTask(
 		return false, err
 	}
 
+	// get the domainName
+	domainName, err := e.domainCache.GetDomainName(domainID)
+	if err != nil {
+		return false, err
+	}
+
+	// tell if the tasklist is partitioned or sticky
```

Comment:
could you add one more line of comment here to say the purpose so other people can know why you emit the metrics here, just like the summary you added for the pr

Reply:
Resolved in newest commit